package.json

```diff
@@ -161,6 +161,7 @@
 		"html-webpack-plugin": "^3.2.0",
 		"jest": "^24.8.0",
 		"jest-transform-css": "^2.0.0",
+		"mongodb-memory-server": "^5.1.10",
```

Comment:
probably need to add `mongodb-memory-server-global` or `mongodb-memory-server-core` to dev deps

Reply:
Actually mongodb-memory-server fulfills the auto-download on npm i.